app/models/user.rb

```diff
@@ -53,4 +50,15 @@ def update_password_and_token(password)
     self.token = nil
     self.save
   end
+
+  def connect_with_invitors
+    invites = Invite.where(email: self.email)
+    if invites.exists?
+      invitor_ids = invites.map(&:user_id).uniq
```

Comment:
You can also use `pluck` instead of `map`, which is more performant since it doesn't pull all of the records into memory to get the user id. You can see more [here](http://www.webascender.com/Blog/ID/553/Rails-Tips-for-Speeding-up-ActiveRecord-Queries#.VTBnc61Vikp).